src/metamodels/crosscat.py

```diff
@@ -246,9 +244,10 @@ def __init__(self, crosscat, subsample=None, ccargs=None,
                  cckwargs=None):
         if isinstance(crosscat, EngineTemplate):
             self._crosscat = crosscat
-            if (ccargs, cckwargs) != (None, None):
-                raise ValueError('ccargs, cckwargs are only for when you'
-                                 'pass a constructor.')
+            if ccargs is not None or cckwargs is not None:
```

Comment:
No... I removed them in a subsequent commit.  Sorry about leaving the confusing history around.  I will try to do better next time.